quartz/components/scripts/search.inline.ts

```diff
@@ -405,11 +406,32 @@ document.addEventListener("nav", async (e: CustomEventMap["nav"]) => {
 
     let searchResults: FlexSearch.SimpleDocumentSearchResultSetUnit[]
     if (searchType === "tags") {
```

Comment:
let's make this a bit easier to read, we can check for the index of the first " " and see if it exists

- if it does, we know we are in the combined search mode (and we can get the query string by substringing `currentSearchTerm` by the index of the first space + 1)
- if it doesn't, we are in the pure tag search mode

Reply:
agreed, changed the code using indexOf and substring